OlympusDAO-Agent-Suit/TotalReserves/src/findings.ts

```diff
@@ -0,0 +1,59 @@
+import { 
+  Finding, 
+  FindingSeverity, 
+  FindingType,
+} from 'forta-agent';
+import { BigNumber } from 'ethers';
+
+const reserveIncreasement = (
+  initialReserve: BigNumber, 
+  curReserve: BigNumber, 
+  percentChange: number
+): Finding => Finding.fromObject({
+  name: "OlympusDAO Treasury totalReserves monitor",
+  description: "Big increasement detected",
+  alertId: "olympus-treasury-3-1",
+  severity: FindingSeverity.Info,
+  type: FindingType.Info,
+  metadata: {
+    totalReserve: initialReserve.toString(),
+    changedTo: curReserve.toString(),
+    increasedBy: `${percentChange}% or more`,
```

Comment:
I think having this in the metadata is not very good, this is a value set in the agent at publishing time and it doesn't change. I think it would be more useful to show the exact percent or not show it at all. For indicating the agent threshold the Description can be used.